libyear/pypi.py

```diff
@@ -65,12 +68,13 @@ def get_version_release_dates(name, version, version_lt):
 
     version = get_version(pypi_data, version)
     if version is None:
+        logging.warning(f'No Pypi version "{version}" for package "{name}"')
         return None, None, None, None
 
     try:
         latest_version_date = releases[latest_version][-1]['upload_time_iso_8601']
     except IndexError:
-        print(f'Latest version of {name!r} has no upload time.')
+        logging.info(f'Latest version of {name!r} has no upload time.')
```

Comment:
Does this cause a package to be skipped? If it does, then maybe we should use `warning`  to call our attention that a pkg is being skipped

Reply:
I hesitated with a warning for all except. I can do the switch if it makes more sense.